ultradns/client.py

```diff
@@ -75,10 +83,12 @@ def is_auth_token_expired(self, response):
         """Return True if response says that the authentication token
         is expired.
         """
-        json_body = response.json()
-        return (response.status_code == requests.codes.UNAUTHORIZED and
-                isinstance(json_body, dict) and
-                json_body.get('errorCode') == ERR_CODE_AUTH_TOKEN_EXPIRED)
+        if response.status_code == requests.codes.UNAUTHORIZED:
+            json_body = response.json()
```

Comment:
Right. I just wanted to have as much context in it is possible, so I pass whole response object.